pkg/importer/dataStream_test.go

```diff
@@ -267,6 +280,13 @@ func Test_dataStream_Close(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
+			buf := make([]byte, 32*1024)
+			for {
+				n, err := ds.Read(buf)
```

Comment:
may still want to use ReadAll in case the reader is a multi-reader.

Reply:
There is no ReadAll defined on a DataStream, just Read (AFAIC).

Reply:
Correct. ds.Read calls ReadAll so we're good here.